hoomd/mpcd/module.cc

```diff
@@ -164,9 +167,18 @@ PYBIND11_MODULE(_mpcd, m)
     mpcd::detail::export_VirtualParticleFiller(m);
     mpcd::detail::export_SlitGeometryFiller(m);
     mpcd::detail::export_SlitPoreGeometryFiller(m);
+    // For slit and slit-pore geometries, manual fillers work better. Therefore we keep them.
+    // However, for the sake of generality of operation, we also keep the option for switching to rejection filler
+    // available.
+    mpcd::detail::export_RejectionVirtualParticleFiller<mpcd::detail::SlitGeometry>(m);
+    mpcd::detail::export_RejectionVirtualParticleFiller<mpcd::detail::SlitPoreGeometry>(m);
```

Comment:
Do not export classes that aren't used, somebody else can come add this back if they want to (it's a one-liner).